src/app/settings/organization-plans.component.html

```diff
@@ -158,13 +158,20 @@ <h2 class="spaced-header">{{'summary' | i18n}}</h2>
             <label class="form-check-label" for="interval{{selectablePlan.type}}">
                 <ng-container *ngIf="selectablePlan.isAnnual">
                     {{'annually' | i18n}}
-                    <small *ngIf="selectablePlan.basePrice">
+                    <small *ngIf="selectablePlan.basePrice && !acceptingSponsorship">
                         {{'basePrice' | i18n}}: {{ selectablePlan.basePrice / 12 | currency:'$'}} &times; 12
                         {{'monthAbbr' | i18n}}
                         =
                         {{selectablePlan.basePrice | currency:'$'}}
                         /{{'year' | i18n}}
                     </small>
+                    <small *ngIf="acceptingSponsorship">
+                        {{'basePrice' | i18n}}: {{ selectablePlan.basePrice / 12 | currency:'$'}} &times; 12
+                        {{'monthAbbr' | i18n}}
+                        =
+                        <span style="text-decoration: line-through;">{{selectablePlan.basePrice | currency:'$'}}</span>
+                        {{'freeWithSponsorship' | i18n}}
+                    </small>
```

Comment:
Can't we just have one `<small>` node and alter the right hand side of the equals depending on the `acceptingSponsorship` boolean?

Reply:
Switched to that.